crates/sui-core/src/tbls/tbls_ids.rs

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use itertools::Itertools;
+use std::collections::HashMap;
+use std::num::NonZeroU32;
+use sui_types::base_types::AuthorityName;
+use sui_types::committee::StakeUnit;
+
+const MAX_NUM_OF_SHARES: u32 = 1000;
+
+type TBlsId = NonZeroU32;
+
+pub struct TBlsIds {
```

Comment:
I would definitely like to see a doc comment here indicating what this is meant for, and a link to the GH issues that will add functionality to this.